patches/tModLoader/Terraria/Social/Steam/SteamedWraps.cs

```diff
@@ -231,9 +233,11 @@ public static EUGCQuery CalculateQuerySort(QueryParameters qParams)
 			return EUGCQuery.k_EUGCQuery_RankedByTextSearch;
 
 		return (qParams.sortingParamater) switch {
+			ModBrowserSortMode.Hot when qParams.days == 0 => EUGCQuery.k_EUGCQuery_RankedByVote, // Corresponds to "Most Popular" for "All Time" on workshop website
 			ModBrowserSortMode.DownloadsDescending => EUGCQuery.k_EUGCQuery_RankedByTotalUniqueSubscriptions,
-			ModBrowserSortMode.Hot => EUGCQuery.k_EUGCQuery_RankedByPlaytimeTrend,
```

Comment:
No longer any filter for `k_EUGCQuery_RankedByPlaytimeTrend`?

Reply:
`Hot` is now using `k_EUGCQuery_RankedByTrend`, which is votes. `Hot`/`k_EUGCQuery_RankedByPlaytimeTrend` was virtually indistinguishable from `k_EUGCQuery_RankedByTotalUniqueSubscriptions`/`DownloadsDescending` , so it didn't really add anything.

Also, Votes (`k_EUGCQuery_RankedByPlaytimeTrend`) for 1 Week and the other time periods shows up-and-coming mods based on the time period. `k_EUGCQuery_RankedByPlaytimeTrend` for different time periods is basically all the same results, so it would too be useless.